openquake/engine/engine.py

```diff
@@ -390,14 +391,20 @@ def job_from_file(cfg_file, username, log_level='info', exports='',
     # read calculation params and create the calculation profile
     params = readinput.get_params([cfg_file])
     params.update(extras)
-    # build and validate an OqParam object
-    oq = readinput.get_oqparam(params)
-    # create the current job
+
+    class oq:  # fake OqParam object
```

Comment:
There isn't any other way to get only these params instead of mocking the class?